tests/foreman/ui/test_sync.py

```diff
@@ -102,6 +94,7 @@ def test_positive_sync_custom_ostree_repo(session, module_custom_product, module
 @pytest.mark.run_in_one_thread
 @pytest.mark.tier2
 @pytest.mark.upgrade
+@pytest.mark.stubbed
```

Comment:
why marking automated test as stubbed?

Reply:
Because we want to skip them until the ostree is supported again.

Reply:
Its wrong approch to skip as it'll be considered as manual test, instead if you've any jira for adding that support again then I'd suggest using skipif open to skip this tests

Reply:
Unfortunately there isn't such Jira nor plan AFAIK, yet I'd prefer keeping the cases than complete remove.

We already have some stubbed out cases (and even whole modules [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_ostreebranch.py#L27) or [there](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_contentview_old.py#L53)) and some other cases [commented out](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_repository.py#L1983) (which I don't like even more).

Are you aware of better ways of not executing the tests but keeping them? Maybe `@pytest.mark.skipif(True, reason='Unsupported right now')`, but it's just another wording.